apps/web/src/studio/LocalStudioAuthenticator.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect } from 'react';
+import { getToken, useAuth } from '../hooks/useAuth';
+import { ROUTES } from '../constants/routes';
+import { encodeBase64 } from './utils/base64';
+import { ILocalStudioState } from './studio.types';
+import { useLocation } from 'react-router-dom';
+
+const BRIDGE_ENDPOINT_PATH = '/api/novu';
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Thank you for extracting this as a constant! Would it perhaps make sense as an env var?‏